prow/jobs_config/knative-extensions/eventing-kafka-broker.yaml

```diff
@@ -46,6 +46,22 @@ jobs:
       - name: BROKER_CLASS
         value: KafkaNamespaced
 
+  - name: reconciler-tests-loom
+    types: [presubmit]
+    command: [runner.sh, ./test/presubmit-tests.sh, --run-test, "./test/reconciler-tests-loom.sh"]
+    requirements: [docker]
+    env:
+      - name: BROKER_CLASS
+        value: Kafka
+
+  - name: reconciler-tests-namespaced-broker-loom
+    types: [ presubmit ]
+    command: [ runner.sh, ./test/presubmit-tests.sh, --run-test, "./test/reconciler-tests-loom.sh" ]
+    requirements: [ docker ]
+    env:
+      - name: BROKER_CLASS
+        value: KafkaNamespaced
```

Comment:
As we discussed, we should make this job optional until we know it works reliably and consistently.

This is the the way to make a job optional https://github.com/knative/infra/blob/c6a27d4bf0c6a19abe4c67ec69a98427ac89c4ae/prow/jobs_config/knative-extensions/eventing-kafka.yaml#L22

Reply:
I have changed it accordingly, let see first if this reconciler tests works or not.